pkg/util/containers/filter_test.go

```diff
@@ -400,16 +425,16 @@ func TestNewMetricFilterFromConfig(t *testing.T) {
 	f, err := newMetricFilterFromConfig()
 	require.NoError(t, err)
 
-	assert.True(t, f.IsExcluded("dd-152462", "dummy:latest", ""))
-	assert.False(t, f.IsExcluded("dd-152462", "apache:latest", ""))
-	assert.False(t, f.IsExcluded("dummy", "dummy", ""))
-	assert.True(t, f.IsExcluded("dummy", "k8s.gcr.io/pause-amd64:3.1", ""))
-	assert.True(t, f.IsExcluded("dummy", "rancher/pause-amd64:3.1", ""))
+	assert.True(t, f.IsExcluded(nil, "dd-152462", "dummy:latest", ""))
```

Comment:
Would it make sense to add tests for IsExcluded when the first argument is not nil?

Reply:
We have other tests covering cases when the first argument is not `nil`. I believe they might be enough.